hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -5943,6 +5943,8 @@ LocatedBlock bumpBlockGenerationStamp(ExtendedBlock block,
     }
     // Ensure we record the new generation stamp
     getEditLog().logSync();
+    LOG.info("bumpBlockGenerationStamp({}, client={}) success",
+        locatedBlock.getBlock(), clientName);
```

Comment:
@tomscut We record block information, will there be a lot of logs? Is it changed to debug?

Reply:
@slfan1989 hi, thanks for your review,  the frequency of bumpBlockGenerationStamp logs is approximately equal to the frequency of  updatePipeline.  So, i think we can use INFO level here.

Reply:
> @tomscut We record block information, will there be a lot of logs? Is it changed to debug?

I was worried about this before, but through offline discussions and testing with @hfutatzhanghb , we found that the logs were not too frequent.

Reply:
@hfutatzhanghb @tomscut Thanks for the information!